docs/installation.rst

```diff
@@ -0,0 +1,386 @@
+
+.. contents::
+
+Installing MUSE
+===============
+
+Recommended installation
+------------------------
+
+To help you installing MUSE in your system we will follow these steps:
+
+- `Launching a terminal`_: Needed to both install and run MUSE.
+- `Installing a compatible Python version`_: At the moment, MUSE works with Python 3.8 and 3.9.
+- `Installing pipx`_: A Python application manager that facilitates installing, keeping applications updated and run them in their own isolated environments.
+- `Installing MUSE itself`_
+
+In the following sections, we will guide you step by step in configuring your system so it can run MUSE. At the end, we include alternative ways of installing MUSE if this method does not work for any reason, for advanced users and for developers of MUSE.
+
+.. note::
+
+    The next sections will explain in detail the appropriate steps as well as commenting on possible caveats, but in the end, what we are going to do to install MUSE is the following:
+
+    - Install `pyenv <https://github.com/pyenv/pyenv>`_ (Linux and MacOS) or `pyenv-win <https://pyenv-win.github.io/pyenv-win/>`_ (Windows) and make sure it works.
+    - Run the following commands in the terminal:
+
+        .. code-block::
+
+            pyenv install 3.9.13
+            pyenv shell 3.9.13
+            python -m pip install pipx
+            python -m pipx ensurepath
+            python -m pipx install muse-os
+
+    - After this, MUSE will be available to use system wide symply by invoking ``muse`` in the terminal.
+
```

Comment:
That will work only if you activate whatever relevant environment you have used to install muse (as it was done until now). Otherwise, it will not. The beauty of this method is that user will not need to worry about activating any environment. 

OK to adding an example of usage.

Reply:
I mean pyenv works nicely overall.
My problem is that many would look only at these lines and will come up with several questions. Can you also add a bullet point on launching the command line?

Reply:
Sure! I can also re-order the text and put the warning about reading through the documentation at the beginning rather than at the end, to make sure people gets it.

Reply:
Fixed in bbad461